server/src/main/java/org/candlepin/model/Content.java

```diff
@@ -528,6 +528,21 @@ public boolean equals(Object other) {
 
     @Override
     public int hashCode() {
+        int hash = 7;
+
+        hash = 17 * hash + (this.id != null ? this.id.hashCode() : 0);
```

Comment:
Could, I suppose. Was just setting up for what I feel is inevitable change on that method.